### PR TITLE
[JARS-118] Handle duplicate filenames

### PR DESCRIPTION
**Pull request by hashknot**
Filenames can be repeated while exporting resources
in a dataset. Ensure both the files are present in the
exported dataset zip file.

### CODE REVIEW

**Comment by hashknot** [comment]
> for some reason, I get a bunch of giles exceptions that the token is invalid (it does download some of them though). This might be a different issue though?
> oh, I just realized that's coming from a different ticket, doesn't it?

That issue was observed when one tried creating a dataset in Amphora without having an active account on Giles. I am not observing this issue locally.

**Comment by jdamerow** [comment]
Hm, that can’t be the issue though, since some of the files get downloaded ok. And I have an active account.

---
Julia Damerow, PhD
Scientific Software Engineer
ASU-SFI Center for Biosocial Complex Systems
Digital Innovation Group
Arizona State University
email: jdamerow@asu.edu <mailto:jdamerow@asu.edu> <>
web: http://diging.asu.edu/ <http://diging.asu.edu/> <>

**Comment by hashknot** [comment]
Didn't you get all the files you requested for in the dataset? Can you give me any pointers on how can I reproduce this issue locally?

**Comment by jdamerow** [comment]
Never mind, my fault. Wrong Giles instance… 
---
Julia Damerow, PhD
Scientific Software Engineer
ASU-SFI Center for Biosocial Complex Systems
Digital Innovation Group
Arizona State University
email: jdamerow@asu.edu <mailto:jdamerow@asu.edu> <>
web: http://diging.asu.edu/ <http://diging.asu.edu/> <>

**Comment by hashknot** [comment]
- Fixed problems with exported images
- Added support for exporting collections as a whole along with nested sub-collections.
- Now displaying content-type instead of "None" in resource title.

File: cookies/accession/giles.py [hunk 1]
```diff
@@ -56,7 +56,11 @@ def get(self, target, raw=False):
         """
         """
         headers = self.build_auth_headers()
-        response = requests.get(self.sign_uri(target), headers=headers)
+
+        # Giles doesn't return images if 'dw' parameter is missing
+        params = {'dw': 500}
```
Comment (jdamerow) [inline cookies/accession/giles.py]: we probably want this to be configurable during export, but I'll create a new ticket for that.

**Comment by jdamerow** [comment]
one last thing, and then we should be good to go: let's add the collection name and id to which a resource belongs to the index.csv file.

**Comment by hashknot** [comment]
Collection ID, name are now included in index.csv file.

**Pull request merged**